starter-code/javascript/chronometer.js

```diff
@@ -1,39 +1,52 @@
 // Constructor
-// function Chronometer() {
-
-// }
-
-// Chronometer.prototype.startClick = function () {
-
-// };
-
-// Chronometer.prototype.setMinutes = function () {
-  
-// };
-
-// Chronometer.prototype.setSeconds = function () {
-  
-// };
-
-// Chronometer.prototype.twoDigitsNumber = function () {
-  
-// };
-
-// Chronometer.prototype.setTime = function () {
-
-// };
+function Chronometer() {
+  this.currentTime = 0;
+  this.intervalId = 0;
+}
+Chronometer.prototype.startClick = function() {
+  var that = this;
+  that.intervalId = setInterval(function() {
+    that.currentTime += 1;
+    that.setTime();
```

Comment:
No tenéis que llamar setTime, tenéis que decir que se ejecute printTime() para que pinte cada segundo el tiempo.